HopsanGUI/HcomHandler.cpp

```diff
@@ -7988,7 +7993,7 @@ bool HcomHandler::evaluateArithmeticExpression(QString cmd)
         evaluateExpression(right);
 
         QStringList pars;
-        if (mAnsType==Scalar)
+        if (mAnsType==Scalar || mAnsType==Wildcard)
```

Comment:
It is unclear to me why this change is needed, what relation does it have with self.
If this is a "Drive by fix" thats fine, I am just curious,

Reply:
Save as above. We need to also evaluate wildcards, in case you want to set a parameter to a string value. Perhaps we should make a separate PR for this, since it can be used for other purposes as well.

Reply:
Ok, keep it